src/ChickenAPI.Packets.Serialization/Deserializers/World/Chat/ClientSayPacketDeserializer.cs

```diff
@@ -0,0 +1,18 @@
+﻿using ChickenAPI.Packets.Game.Client.Chat;
+
+namespace ChickenAPI.Packets.Serialization.Deserializers.World.Chat
+{
+    public class ClientSayPacketDeserializer : GenericBasePacketDeserializer<ClientSayPacket>
+    {
+        protected override string Header => "say";
+
+        protected override ClientSayPacket DeserializeImpl(string buffer)
+        {
+            return new ClientSayPacket
```

Comment:
protected override ClientSayPacket DeserializeImpl(string buffer) => new ClientSayPacket {}

Reply:
This was written like that for potential future case of check implementations (length of the buffer, special characters...)